client/my-sites/site-settings/form-general.jsx

```diff
@@ -721,7 +721,7 @@ export class SiteSettingsFormGeneral extends Component {
 					isSaving={ isSavingSettings }
 					onButtonClick={ handleSubmitForm }
 					showButton
-					title={ translate( 'Site Profile' ) }
+					title={ translate( 'Site profile' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.